### PR TITLE
Add new translations in BlazePress

### PR DESCRIPTION
**Pull request by jjolmo**
Update translations

Related to #

## Proposed Changes

* New translations. Generated the file through the DSP

image

image

image

image

image

image

image

image

image

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9287302

Thank you @jjolmo for including a screenshot in the description! This is really helpful for our translators.

File: client/lib/promote-post/string.ts [hunk 1]
```diff
@@ -9,21 +9,17 @@ const BlazePressStrings = () => {
...
+	translate( 'Learn more about the {{linkAdvertisingPolicy}}Advertising Policy{{/linkAdvertisingPolicy}}.' );
 	translate( 'Creating campaign…' );
 	translate( 'Make the most of your Blaze campaign' );
 	translate( 'Choose an eye-catching image for your ad' );
 	translate( 'Adjust your title to make it more engaging' );
 	translate( 'Pick the right audience, budget and duration' );
 	translate( 'Get started' );
 	translate( 'Learn more' );
-	translate( "Don't show me this step again." );
+	translate( 'Don\'t show me this step again.' );
```
Comment (timur987) [inline client/lib/promote-post/string.ts]: Should we use another apostrophe ("`’`") as above instead?

File: client/lib/promote-post/string.ts [hunk 2]
```diff
@@ -68,56 +62,40 @@ const BlazePressStrings = () => {
...
-	translate(
-		'We created this campaign to deliver the most valuable traffic, yet you can still make changes before submitting it.'
-	);
+	translate( 'We created this campaign to deliver the most valuable traffic, yet you can still make changes before submitting it.' );
 	translate( 'Duration' );
 	translate( 'Budget' );
 	translate( 'day' );
 	translate( 'Estimated impressions' );
 	translate( 'Payment' );
+	translate( 'Loading...' );
```
Comment (timur987) [inline client/lib/promote-post/string.ts]: Here should be ellipsis symbol ("`…`") instead. Here's what TeamCity said:
```
Error: Fixable rules must set the `meta.fixable` property to "code" or "whitespace".
Occurred while linting /home/teamcity-1/buildAgent/work/c4a9d5b38c1dacde/client/lib/promote-post/string.ts:88
Rule: "wpcalypso/i18n-ellipsis"
```

**Review by timur987 — COMMENTED** [review]
Thanks, @jjolmo!

The changes look good. I still need to figure out how to test them.
Left a couple of suggestions below, one of them is about linter issue in TeamCity.

File: client/lib/promote-post/string.ts (hunk 2 above)
Comment (jjolmo) [reply client/lib/promote-post/string.ts]: oop that's right. I'll change it in DSP code then pass the change to this file

File: client/lib/promote-post/string.ts (hunk 1 above)
Comment (jjolmo) [reply client/lib/promote-post/string.ts]: This is something autogenerated and fixed by eslint rules directly, so not sure if I should. I can change it manually, but it isn't something I did manually

**Comment by jjolmo** [comment]
Thanks Timur for your review! I won't merge this PR until the next week, while I wait for translations to get done

**Review by timur987 — APPROVED** [review]
Thanks @jjolmo. Changes look great.
Have not tested them in Calypso though.

**Review by j6ll — APPROVED** [review]

**Comment by j6ll** [comment]
Changes looks good, just 1 test is failing 👍

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9287302).

**Pull request merged**